inference-engine/src/hetero_plugin/CMakeLists.txt

```diff
@@ -16,8 +16,8 @@ ie_faster_build(${TARGET_NAME}
     UNITY
 )
 
-target_link_libraries(${TARGET_NAME} PRIVATE ade pugixml inference_engine
```

Comment:
Just for education why did we use ade?

Reply:
for graph splitting. Now HETERO has its own split logic